src/ilasm/grammar_after.cpp

```diff
@@ -1500,6 +1501,27 @@ AsmParse::~AsmParse()
     while(m_ANSLast.POP());
 }
 
+#ifdef FEATURE_PAL
+// This is a simplified implementation of IsTextUnicode.
+BOOL IsTextUnicode(CONST VOID* lpv, int iSize, LPINT lpiResult)
+{
+    *lpiResult = 0;
+
+    if (iSize < 2) return FALSE;
+
+    BYTE * p = (BYTE *)lpv;
+
+    // Check for Unicode BOM
```

Comment:
@krwq This is a very limited implementation of the Windows API IsTextUnicode. See 
https://msdn.microsoft.com/en-us/library/windows/desktop/dd318672(v=vs.85).aspx
Hence the name.